uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/DrupalUserServiceTest.php

```diff
@@ -23,8 +23,7 @@ class DrupalUserServiceTest extends CRM_HRCore_Test_BaseHeadlessTest {
   private $testContact;
 
   public function setUp() {
-    CRM_Utils_API_ReloadOption::singleton();
-    $this->loadRequiredData();
+    $this->createUserAndActivities();
     $this->registerCurrentLoggedInContactInSession($this->testContact['id']);
     $this->cleanup();
```

Comment:
Why is the cleanup() inside the `setUp()` step? Usually, we clean things up on the `tearDown()`